frontend/index.html

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
+    <meta name="google" content="notranslate" />
```

Comment:
this tag doesn't prevent google from translating anymore according to this:
https://stackoverflow.com/questions/55954813/meta-name-google-content-notranslate-not-working-anymore
I believe "content" needs to be replaced with "value". But I have not tested it. Will integrate this if you can confirm which one works.

Reply:
Oh, I just copied this from a place where it's in use and working for us. After googling this, the 2021 way to do <html translate="no" according to https://stackoverflow.com/a/12238414/296639
I'll implement this addition also in the other app where I got the old code from :) Thanks!